vue-app/src/components/Cart.vue

```diff
@@ -582,6 +581,26 @@ export default class Cart extends Vue {
     if (!el) return
     el.scrollIntoView({ behavior: 'smooth' })
   }
+
+  splitContributionsEvenly(): void {
+    const { cart, currentRound: { contributions } } = this.$store.state
+    const total = this.$store.getters.canUserReallocate
+      ? contributions._value
```

Comment:
I _think_ this is incorrect.

This `total` should be the sum of the current user's contribution amounts. I believe `currentRound.contributions` is the sum of ALL user contributions on the round: https://github.com/ethereum/clrfund/blob/develop/vue-app/src/api/round.ts#L149

I think the correct way for us to calculate this is the same reduce approach you took below on `cart` but with `committedCart` instead.

Reply:
Im not sure about that. Here was my thinking and why I did it this way:
- User contributes 50 DAI to 2 projects, 100 DAI total.
- User edits these two projects to have 45 DAI. 90 DAI to projects, 10 DAI to matching pool, 100 DAI total. This is then reallocated.
- User goes back to reallocate and splits evenly, would want to split the total contributed on reallocation here, which would be 100 DAI.

But, im not against what you're saying here. This is just my thoughts on it and why I did it this way. If you do think it should be the committedCart total instead of total contribution then will change this.

Reply:
I agree with that, which yes does maybe make adding the amount of the `commitedCart` problematic. My main point though is the `contributions` value you're using now is pulling the total contributions of ALL contributors of the current round. We definitely don't want to split on that value 😄

Reply:
If I'm correct on that then the question becomes: do we need to store additional info about the committed matching pool amount in gun db? Or is it possible to derive that value from elsewhere?

Reply:
It looks like we may have that via `contribution` on the store (vs. `currentRound.contributions`).

Reply:
Just so im understanding the case your talking about, if a user were to just add funds to the matching pool in addition to doing the contribution then this would be included in that?

Reply:
Circling back from our call - looks like we can indeed use the `contribution` value from the store.